platform/viewer/src/connectedComponents/ViewerRetrieveStudyData.js

```diff
@@ -340,6 +341,10 @@ function ViewerRetrieveStudyData({
         }
       }
 
+      if (appConfig.enableGoogleCloudAdapter) {
+        retrieveParams.push(true); // Seperate SeriesInstanceUID filter calls.
```

Comment:
This is really weird. Let's hope we never reorganize the arguments to the function.

Reply:
I agree, I'd rather have a deconstructed object inside `retrieveStudiesMetadata`, but it'd change what is actually top level API in `@ohif/core`.